viz_scripts/generic_metrics.ipynb

```diff
@@ -121,11 +121,16 @@
     "file_name='ntrips_mode_confirm%s' % file_suffix\n",
     "plot_title_no_quality= \"Number of trips for each mode (selected by users)\"\n",
     "try:\n",
+    "  \n",
     "    labels_mc = expanded_ct['Mode_confirm'].value_counts(dropna=True).keys().tolist()\n",
-    "    values_mc = expanded_ct['Mode_confirm'].value_counts(dropna=True).tolist()    \n",
+    "    values_mc = expanded_ct['Mode_confirm'].value_counts(dropna=True).tolist()\n",
+    "    \n",
+    "    data = pd.DataFrame({\"Mode_confirm\": labels_mc,\n",
+    "                   \"Number of Trips\": values_mc})\n",
```

Comment:
I don't see why you have this copy-pasted before every line.
Can you please explain:
- why we need to create a separate data frame for the visualization of bar charts when we did not for pie charts?
- why this needs to be copy-pasted instead being refactored into common code with the same interface?